src/app/organization/[orgSlug]/admin/pending-invites.tsx

```diff
@@ -38,30 +39,35 @@ const PendingUser: React.FC<{ orgSlug: string; pending: { id: string; email: str
     )
 }
 
-export const PendingUsers: FC<PendingUsersProps> = ({ orgSlug }) => {
+export const PendingUsers: FC<PendingUsersProps> = ({ orgSlug, isVisible }) => {
     const { data: pendingUsers = [], isLoading: isLoadingPending } = useQuery({
         queryKey: ['pendingUsers', orgSlug],
         queryFn: () => getPendingUsersAction({ orgSlug }),
     })
 
-    return (
-        <div data-testid="pending-invites">
-            <Text fw={600} mb="md">
-                Pending invitations
-            </Text>
-            <LoadingMessage isVisible={isLoadingPending} message="Loading pending invitations…" />
+    if (!isVisible) return null
 
-            {!pendingUsers.length && (
-                <Text size="sm" c="dimmed">
-                    No pending invitations for this organization.
+    return (
+        <>
+            <Divider c="charcoal.1" my="xl" />
+            <div data-testid="pending-invites">
+                <Text fw={600} mb="md">
+                    Pending invitations
                 </Text>
-            )}
+                <LoadingMessage isVisible={isLoadingPending} message="Loading pending invitations…" />
```

Comment:
Same logic applies here, `LoadingMessage` could drop the `isVisible` prop and you can just conditionally render wherver you're using it like

```
isLoading && <LoadingMessage message="Loading pending invitations…" />
```